src/App.js

```diff
@@ -1,24 +1,20 @@
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+import Header from './components/Header';
+import Books from './components/pages/Books';
+import Categories from './components/pages/Categories';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Header />
+      <Routes>
+        <Route path="Books" element={<Books />} />
```

Comment:
- [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page, so please can you set the Books component as the default page 🙃 🙃 😁 
eg:
```jsx
    <Route exact path="/">
      <Books />
    </Route>
```